mne/decoding/time_gen.py

```diff
@@ -449,6 +449,8 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode):
         elif predict_mode == 'cross-validation':
             # Predict with the estimator trained on the separate training set.
             for k, (train, test) in enumerate(cv):
+                if not np.any(test):
```

Comment:
Although maybe the check here is too lenient. In other words, `test` could be non-empty and all zeros and this will evaluate to go to the `continue`, which isn't right I don't think. Maybe you want to do `if np.asarray(test).size == 0`?